extensions/cornerstone/src/services/ViewportService/Viewport.ts

```diff
@@ -63,7 +63,7 @@ export type PublicDisplaySetOptions = {
   blendMode?: string;
   slabThickness?: number;
   colormap?: string;
-  presetName?: string;
+  displayPreset?: string;
```

Comment:
Does this need to be changed and/or added in any of the (*.md) docs?

Reply:
I'm pretty sure I updated it everywhere

Reply:
Does it perhaps make sense to update the list of display set options in this section of the docs? https://v3-docs.ohif.org/platform/extensions/modules/hpModule#viewports

![image](https://user-images.githubusercontent.com/109477394/236976546-ab940d87-b75a-41d9-983f-331510e7d58a.png)